recipes/SpatialPy/LICENSE

```diff
@@ -0,0 +1,674 @@
+GNU GENERAL PUBLIC LICENSE
```

Comment:
No need to include the license file here manually, since it's already included in the PyPI tarball.